app/models/replacement_document_validation_request.rb

```diff
@@ -8,7 +8,7 @@ class ReplacementDocumentValidationRequest < ValidationRequest
   delegate :invalidated_document_reason, to: :old_document
   delegate :validated?, :archived?, to: :new_document, prefix: :new_document
 
-  has_one :new_document, as: :owner, class_name: "Document"
+  has_one :new_document, as: :owner, class_name: "Document" # rubocop:disable Rails/HasManyOrHasOneDependent
```

Comment:
This one as well - planning application has `dependent: :destroy` on `has_many :validation_requests`